wordpress-mobile/WordPress-Android
# [Reader Improvements] Update following status button design

Fixes #19260 

### Do not merge
Should only be merged after https://github.com/wordpress-mobile/WordPress-Android/pull/19281

Fix the design of "Following" button status.

Design specs: zQnohyMpLzBzQ5jzMMKni3-fi

To test:
1 - Run JP and sign in;
2 - Open debug settings and enable `ReaderImprovementsFeatureConfig` FF;
3 - Open reader;
4 - Verify the follow/unfollow button design using dark/light themes in "Sites to follow",  "You might like" tags, post header, site header and tag header: the design should be correct. Make sure that the follow/unfollow action still works as expected;
5 - Repeat step 2 but this time disable the FF;
6 - Repeat steps 3-4 and verify the layout: the follow icon should appear like it currently does in prod;

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Changes were only made to views

PR submission checklist:


UI Changes testing checklist:
